### DISCUSSION TITLE
Project Rebranding to OWASP MAS

### DISCUSSION THREAD

**Discussion by cpholguera** [OWASP/mastg#2180] [Announcements]
Until now our project was called the “OWASP Mobile Security Testing Guide (MSTG)” project. Unfortunately, this was a source of confusion since we happen to have a resource with the same name, the OWASP MSTG. Not only that, that name doesn’t reflect the full scope and reach of our project. Have you ever wondered why the MSTG is called MSTG and not MASTG? Both documents are about Mobile Application Security and we'd like to make that clear.

Today we are rebranding our project to “OWASP Mobile App Security (MAS)”. The OWASP MAS project includes:

- OWASP MASVS -> MAS Verification Standard
- OWASP MASTG -> MAS Testing Guide
- OWASP MAS Checklists
- OWASP MAS Crackmes (incl. Hacking playground)

![mas-rebranding](https://user-images.githubusercontent.com/29175115/186138765-41ea7170-965e-4947-bd58-c00d1a40e581.png)

We see MAS reflecting all the consistency, structure and transparency that we’re bringing with our 2.0 versions.

**The rebranding will happen gradually so expect changes to be rolled out over the upcoming days/weeks.**

**Comment by fbaptista** [comment]
This is actually a great improvement to avoid confusions. Thank you!

**Reply by cpholguera** [reply]
Thank you @fbaptista, we really appreciate your feedback and hope you enjoy using the project.
